functions/Set-DbaAgReplica.ps1

```diff
@@ -177,6 +194,14 @@ function Set-DbaAgReplica {
                         $agreplica.SeedingMode = $SeedingMode
                     }
 
+                    if ($ReadOnlyRoutingList) {
+                        $rorl = [List[IList[string]]]::new();
```

Comment:
ah, thank you for highlighting this! we don't accept `::new` because it's not v3 compat. So this would have to be some sort of ...array? $rorl = @() may suffice.